cmd/kubelet/app/options/options.go

```diff
@@ -98,6 +98,8 @@ func (s *KubeletServer) AddFlags(fs *pflag.FlagSet) {
 	fs.DurationVar(&s.HTTPCheckFrequency.Duration, "http-check-frequency", s.HTTPCheckFrequency.Duration, "Duration between checking http for new data")
 	fs.StringVar(&s.ManifestURL, "manifest-url", s.ManifestURL, "URL for accessing the container manifest")
 	fs.StringVar(&s.ManifestURLHeader, "manifest-url-header", s.ManifestURLHeader, "HTTP header to use when accessing the manifest URL, with the key separated from the value with a ':', as in 'key:value'")
+	fs.MarkDeprecated("manifest-url-header", "Use --manifest-url-headers instead. Will be removed in a future version.")
```

Comment:
Use string array var and specify multiple times. We don't want to use stringslicevar with data that could contain commas